src/lv_conf.h

```diff
@@ -534,8 +534,9 @@
  *  E.g. __attribute__((aligned(4)))*/
 #define LV_ATTRIBUTE_MEM_ALIGN
 
-/** Attribute to mark large constant arrays, for example for font bitmaps */
-#define LV_ATTRIBUTE_LARGE_CONST
+/*Attribute to mark large constant arrays for example font's bitmaps*/
+/* Here we set all large const array in the graphical_assests section (in OSPI) */
+#define LV_ATTRIBUTE_LARGE_CONST BSP_PLACE_IN_SECTION(".graphical_assets")
```

Comment:
Don't set it by default as it affects the performance, but describe it in the README instead.